From 17a067b9b823a0d54e061eae45ff8e2c7e4a88d0 Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Fri, 21 Jan 2022 08:32:31 +0100
Subject: [PATCH] build: Remove incorrect i18n.merge_file argument

The positional argument was being silently ignored until meson 0.60.0 where
it fails with "ERROR: Function does not take positional arguments".

Related: https://github.com/mesonbuild/meson/issues/9441
Fixes: https://gitlab.gnome.org/GNOME/gvfs/-/issues/599

[Retrieved (and backported) from:
https://gitlab.gnome.org/GNOME/gvfs/-/commit/17a067b9b823a0d54e061eae45ff8e2c7e4a88d0]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 daemon/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/daemon/meson.build b/daemon/meson.build
index 5f9559aa..e20ada29 100644
--- a/daemon/meson.build
+++ b/daemon/meson.build
@@ -360,7 +360,6 @@ if enable_admin
   )
 
   i18n.merge_file(
-    policy,
     input: policy_in,
     output: '@BASENAME@',
     po_dir: po_dir,
-- 
GitLab

